extensions/resteasy-reactive/rest-client-reactive/deployment/src/main/java/io/quarkus/rest/client/reactive/deployment/RestClientReactiveProcessor.java

```diff
@@ -580,6 +573,28 @@ && isImplementorOf(index, target.asClass(), RESPONSE_EXCEPTION_MAPPER, Set.of(AP
         }
     }
 
+    /**
+     * Based on a list of interfaces implemented by @Provider class, determine if registration
+     * should be skipped or not. Server-specific types should be omitted unless implementation
+     * of a <code>ClientRequestFilter</code> exists on the same class explicitly. 
+     * Features should always be omitted.
+     */
+    private boolean skipAutoDiscovery(List<DotName> providerInterfaceNames) {
```

Comment:
```suggestion
    private boolean skipAutoDiscoveredProvider(List<DotName> providerInterfaceNames) {
```
...makes it more clear in the context of the larger processor class.

Reply:
Indeed, more concise.